app/developers/commands.py

```diff
@@ -179,3 +179,43 @@ def seed_grants() -> None:
 
     db.session.commit()
     click.echo(f"Loaded/synced {len(export_data['grants'])} grant(s) into the database.")
+
+
+@developers_blueprint.cli.command(
+    "seed-grants-many-submissions", help="Load grants with 100 random submissions into the database"
+)
+def seed_grants_many_submissions() -> None:
```

Comment:
nice - this feels like a really handy thing to have locally so we can start caring about, and validating, some of the performance characteristics 🙇